unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -132,23 +135,38 @@ void onFailWrapper(bool isFatalError)
 
         public void InitializeImpostor()
         {
+            if (impostorInitialized)
+                return;
```

Comment:
Apparently the fetched users snapshot is not being applied to the impostor, I suspect this escape check could be the culprit, do you want me to give it a try at fixing it?

https://user-images.githubusercontent.com/1031741/134537206-4fd0de95-3177-47cd-aee3-7de6cd82f4f4.mov